api/nodes_test.go

```diff
@@ -60,7 +60,7 @@ func TestNodes_PrefixList(t *testing.T) {
 			return false, fmt.Errorf("expected 1 node, got: %d", n)
 		}
 		nodeID = out[0].ID
-		dc = out[0].Datacenter
+		_ = out[0].Datacenter
```

Comment:
What is the need of this statement if we are not using the value of `out[0].Datacenter`?